domain/model/state/controllerstate.go

```diff
@@ -1142,10 +1178,9 @@ AND owner_name = $dbCloudOwner.owner_name
 	)
 	err = db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
 		var result dbCloudCredential
-		if err := tx.Query(ctx, stmt, args).Get(&result); errors.Is(err, sqlair.ErrNoRows) {
-			return fmt.Errorf("%w for name %q and owner %q", modelerrors.NotFound, modelName, modelOwnerName)
-		} else if err != nil {
-			return fmt.Errorf("getting cloud name and credential for model %q with owner %q: %w", modelName, modelOwnerName, err)
+		err := tx.Query(ctx, stmt, args).Get(&result)
+		if errors.Is(err, sqlair.ErrNoRows) {
```

Comment:
Yes, we can get away with not checking err == nil but it's not as readable as the reader has to wonder if it's a mistake or not etc.